pages/workspace/tracking-plan/publishing.mdx

```diff
@@ -139,6 +141,27 @@ Avo will not impact any events or properties that are only defined in Govern, or
   style={{ margin: '32px 0 -32px 0' }}
 />
 
+### Govern Publishing Errors
```

Comment:
The image above this title belongs to the webhook section below—all this new content should probably be above it.

Reply:
Fixed in 0e9b533 ✨